integration-tests/src/user/mod.rs

```diff
@@ -262,10 +262,19 @@ pub trait User {
         actions: Vec<Action>,
     ) -> Result<FinalExecutionOutcomeView, ServerError> {
         let inner_signer = create_user_test_signer(&signer_id);
-        let user_nonce = self
-            .get_access_key(&signer_id, &inner_signer.public_key)
-            .expect("failed reading user's nonce for access key")
-            .nonce;
+        let access_key = self.get_access_key(&signer_id, &inner_signer.public_key);
+
+        let user_nonce = match access_key {
+            Ok(access_key) => access_key.nonce,
+            Err(_) => match signer_id.get_account_type() {
+                AccountType::EthImplicitAccount => {
+                    // TODO Integration-tests/MetaTx: What nonce should we use here?
```

Comment:
This is the case when an ETH account was created, but does not have access key added yet.
What nonce should be used by the account owner?